src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ViewCellRenderer.cs

```diff
@@ -47,16 +47,25 @@ protected override AView GetCellCore(Cell item, AView convertView, ViewGroup par
 			var view = (IPlatformViewHandler)cell.View.ToHandler(cell.FindMauiContext());
 			cell.View.IsPlatformEnabled = true;
 
-			ViewCellContainer c = view.PlatformView.GetParentOfType<ViewCellContainer>();
+			// If the convertView is null we don't want to return the same view, we need to return a new one.
+			// We should probably do this for ListView as well
+			if (ParentView is TableView)
```

Comment:
Can we create an issue linking to this comment to remember to review the same changes with the ListView?